README.md

```diff
@@ -78,35 +52,7 @@ Refer to [AST Serialization Media Types](https://github.com/apiaryio/api-bluepri
 	$ make test
 	```
 
-	To include integration tests (using Cucumber) use the `--include-integration-tests` flag: 
-
-	```sh
-	$ ./configure --include-integration-tests
-	$ make test
-	```
-	
 We love **Windows** too! Please refer to [Building on Windows](https://github.com/apiaryio/snowcrash/wiki/Building-on-Windows).
-		
-### Snow Crash command line tool
-1. Build `snowcrash`:
-	
-	```sh
-	$ make snowcrash
-	```
-
-2. Install & use `snowcrash`:
-
-	```sh
-	$ sudo make install
-	$ snowcrash --help
-	```
-
-## Bindings
```

Comment:
But we can revisit the documentation when updating the documentation for both snowcrash and drafter.